src/TextAnalyzer/Form.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+namespace App\TextAnalyzer;
+
+use InvalidArgumentException;
+
+class Form
+{
+    /**
+     * @const string Path to resource folder
+     */
+    protected const RESOURCE_DIR = __DIR__ . DIRECTORY_SEPARATOR . 'resource' . DIRECTORY_SEPARATOR;
+    protected const OUTPUT = 'output';
+
+    /**
+     * @var string
+     */
+    protected $form;
+
+    /**
+     * @var array
+     */
+    protected $options;
+
+    /**
+     * Form constructor.
+     * @param string $form
+     * @param array $options
+     * @throws InvalidArgumentException
+     */
+    public function __construct(string $form, $options = [])
+    {
+        $form = $this->validateForm($form);
+
+        if ($form) {
+            $this->form = $form;
+        }
+
+        if ($this->validateOptions($options)) {
+            $this->options = $options;
+        }
+
+        $this->applyDefaultOptions();
+    }
+
+    /**
+     * @param string $form
+     * @param array|null $options
+     */
+    public function render(string $form = '', ?array $options = null): void
+    {
+        if (isset($options)) {
+            $this->validateOptions($options);
+        }
+
+        $options = $options ?? $this->options;
+
+        if (strlen($form) > 0 && $this->validateForm($form)) {
```

Comment:
having $form and $this->form is confusing, especially with two returns

Reply:
I applied more simpled solution